Source/Public/Get-VSTeamWorkItemTag.ps1

```diff
@@ -0,0 +1,38 @@
+function Get-VSTeamWorkItemTag {
+   [CmdletBinding(HelpUri='https://methodsandpractices.github.io/vsteam-docs/docs/modules/vsteam/commands/Get-VSTeamWorkItemTag')]
+   param (
+
+
+      [Parameter(Mandatory = $true, ValueFromPipelineByPropertyName = $true)]
+      [Alias('TagName')]
+      [Alias('TagId')]
+      [string] $TagIdOrName,
```

Comment:
Question. API says that in the object ID and name are two different properties. But in the URL it does not care if you use the name or the GUID?